tests/global.c

```diff
@@ -0,0 +1,13 @@
+int a = 44;
```

Comment:
How about dropping `tests/global.c` and adding the original test in file `tests/driver.sh`?
I am planning to delete some trivial files in directory `tests` such as `def.c`, `fib.c`, `loop.c`, and `pointer.c`, which are already covered in test driver.

Reply:
However, it still makes sense to generate an executable from `tests/hello.c`.

Reply:
Done! Move global initialization into `tests/driver.sh`